### PR TITLE
fix: Get header version from URL instead of theme.conf

### PR DESCRIPTION
**Pull request by samjwu**
to address https://github.com/ROCm/rocm-docs-core/issues/757

### CODE REVIEW

**Comment by saadrahim** [comment]
How does latest know what version it is?

**Comment by samjwu** [comment]
> How does latest know what version it is?

It grabs it from a file in the header-versions branch https://github.com/ROCm/rocm-docs-core/blob/header-versions/latest_version.txt

File: src/rocm_docs/theme.py [hunk 1]
```diff
@@ -17,6 +17,36 @@
...
+
+def _add_custom_context(
+    app: Sphinx,  # noqa: ARG001
+    pagename: str,  # noqa: ARG001
+    templatename: str,  # noqa: ARG001
+    context: dict[str, str],
+    doctree: object,  # noqa: ARG001
+) -> None:
+    header_latest_version = _get_version_from_url(
+        "https://raw.githubusercontent.com/RadeonOpenCompute/rocm-docs-core/header-versions/latest_version.txt"
```
Comment (saadrahim) [inline src/rocm_docs/theme.py]: change the org to ROCm

File: src/rocm_docs/theme.py [hunk 2]
```diff
@@ -17,6 +17,36 @@
...
+    )
+
+    header_release_candidate_version = _get_version_from_url(
+        "https://raw.githubusercontent.com/RadeonOpenCompute/rocm-docs-core/header-versions/release_candidate.txt"
```
Comment (saadrahim) [inline src/rocm_docs/theme.py]: ROCm

**Review by saadrahim — CHANGES_REQUESTED** [review]
I don't think this approach is suitable for offline environments.

**Comment by samjwu** [comment]
> I don't think this approach is suitable for offline environments.

No, but offline build functionality is also currently blocked by other features such as intersphinx

**Review by yhuiYH — APPROVED** [review]
OK, approved

**Pull request merged**